tracing-subscriber/src/filter/env/mod.rs

```diff
@@ -204,6 +204,22 @@ pub struct EnvFilter {
     regex: bool,
 }
 
+/// Creates an [`EnvFilter`] with the same directives as `self`. This does *not* clone any of the
+/// dynamic state that [`EnvFilter`] acquires while attached to a subscriber.
+impl Clone for EnvFilter {
+    fn clone(&self) -> EnvFilter {
+        EnvFilter {
+            statics: self.statics.clone(),
+            dynamics: self.dynamics.clone(),
+            has_dynamics: self.has_dynamics,
+            by_id: RwLock::new(Default::default()),
```

Comment:
```suggestion
            by_id: RwLock::default(),
```

should work, right? Same below.